smoke-test/tests/cypress/cypress/e2e/incidentsV2/v2_incidents.js

```diff
@@ -141,8 +141,9 @@ describe("incidents", () => {
     })
       .should("be.visible")
       .click();
+    cy.get('[data-testid="filter-base"]').click();
     cy.get('[data-testid="child-option-RESOLVED"]').click();
-    cy.get('[data-testid="nested-options-dropdown-container"]').click();
+    cy.get('[data-testid="filter-base"]').click();
```

Comment:
i saw this test failing not sure if it's related to your changes but this test is also a known flake